### ISSUE TITLE
Workspace * does not exist for prisma seed on Heroku

### ISSUE THREAD

**Issue by divyenduz** [prisma/prisma#2757]
**Describe the bug**
When using Prisma server on Heroku (or custom server on AWS), some commands including `prisma seed`, `prisma seed` do not work and throw the following error. 

```json
ERROR: Workspace * does not exist

{
  "data": {
    "generateClusterToken": null
  },
  "errors": [
    {
      "message": "Workspace * does not exist",
      "locations": [
        {
          "line": 3,
          "column": 9
        }
      ],
      "path": [
        "generateClusterToken"
      ],
      "code": 222
    }
  ],
  "status": 200
}
```

**To Reproduce**
1. Deploy any prisma service on [Heroku](https://www.prisma.io/blog/heroku-integration-homihof6eifi/).
2. Run `prisma seed`

**Expected behavior**
The commands should have the context of workspace and work. Need to test this issue for all commands as it is exhibit only by some commands. 

**Versions (please complete the following information):**
 - `prisma` CLI: `prisma/1.11.1 (darwin-x64) node-v10.4.0`
 - Prisma Server: `1.11`

**Comment by divyenduz** [comment]
I can confirm that issue is only with `seed`, `delete` commands. Rest of the commands work as expected.

**Comment by marktani** [comment]
This is now available at `1.13-beta`.

**Comment by marktani** [comment]
Fixed in `1.13`.
